<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Smartphone Database</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>780</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Введите текст для поиска...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>420</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>460</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Редактировать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>460</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="aboutButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>420</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>О программе</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionCreateFile"/>
    <addaction name="actionSaveFile"/>
    <addaction name="actionLoadFile"/>
    <addaction name="actionMergeFiles"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Редактировать</string>
    </property>
    <addaction name="actionAddSmartphone"/>
    <addaction name="actionEditSmartphone"/>
    <addaction name="actionDeleteSmartphone"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCreateFile">
   <property name="text">
    <string>Создать файл</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="text">
    <string>Сохранить файл</string>
   </property>
  </action>
  <action name="actionLoadFile">
   <property name="text">
    <string>Загрузить файл</string>
   </property>
  </action>
  <action name="actionMergeFiles">
   <property name="text">
    <string>Объединить файлы</string>
   </property>
  </action>
  <action name="actionAddSmartphone">
   <property name="text">
    <string>Добавить смартфон</string>
   </property>
  </action>
  <action name="actionEditSmartphone">
   <property name="text">
    <string>Редактировать смартфон</string>
   </property>
  </action>
  <action name="actionDeleteSmartphone">
   <property name="text">
    <string>Удалить смартфон</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
